src/model/structures/CardAction.cs

```diff
@@ -9,9 +9,12 @@ namespace tcg
   class CardAction : PlayerAction
   {
     public int InHandIndex { get; set; }
-    public CardAction(int playerId, ActionType type, int inHandIndex) : base(playerId, type)
+    public int Power { get; set; }
```

Comment:
Is it only for healing?

Reply:
We need to remove all structures linked to action because we have unified action  now